### PR TITLE
Pricing view

### PR DESCRIPTION
**Pull request by jonastore**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +30,12 @@
     "react-table": "^6.8.0",
     "redux": "^3.7.2",
     "redux-thunk": "^2.2.0",
+    "semantic-ui-css": "^2.3.1",
+    "semantic-ui-react": "^0.79.1",
+<<<<<<< HEAD
```
Comment (SebastianGerS) [inline package.json]: you have to resolve the merge conflict here

File: src/Components/Buttons/GetStarted.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import { Header, Table, Icon, Button } from 'semantic-ui-react'
+
+//onClick={this.redirect('/start') basically linking to the start spage.
+//maybe there's a better way than using this one-line crap
+
+const GetStarted = () => (
+  <div className='ui center aligned basic segment'>
+
+      <Button padded centered size='huge' className='centered'>
```
Comment (SebastianGerS) [inline src/Components/Buttons/GetStarted.js]: padded  and centered  on the button component gives a warning when compiled.
I don't think those values can be associated with the semantic button buttons are always padded if the  compact attribute is not set.

File: src/Components/Buttons/GetStarted.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Header, Table, Icon, Button } from 'semantic-ui-react'
```
Comment (SebastianGerS) [inline src/Components/Buttons/GetStarted.js]: Header, table, icon are defined butt never used, please remove

File: src/Views/Site/Pricing.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import GetStarted from '../../Components/Buttons/GetStarted'
+
+import { Header, Table, Icon, Button } from 'semantic-ui-react'
```
Comment (SebastianGerS) [inline src/Views/Site/Pricing.js]: Button is defined but never used, please remove.

**Review by SebastianGerS — COMMENTED** [review]
Looks good, the responsiveness seams to work now, just some minor things, fix and then push, no need to do a new pull request

**Review by SebastianGerS — APPROVED** [review]
look good!

made a smal commit to adjust the getstarted button, everything else looks great!

**Comment by izabelrosen** [comment]
Good work - but please make the table more centered of the page!

**Pull request merged**